ui/tempo-plugin/src/plugins/tempo-trace-query/DashboardTempoTraceQueryEditor.tsx

```diff
@@ -45,7 +45,7 @@ export function DashboardTempoTraceQueryEditor(props: DashboardTempoTraceQueryEd
         />
       </FormControl>
       <TraceQLEditor
-        completeConfig={{ remote: { url: datasourceURL } }}
```

Comment:
@zhuje I don't know why it was failing here. If you have a quick answer I'd be interested, otherwise I will check deeper.

Reply:
Just from a quick look...I think you need to remove `datasourceURL` from line 30. I haven't tested it by running `npm lint` again  but this is my assumption from a glance.

Reply:
Sure I was meaning why the completeConfig field as is was failing.
I had an error message so I removed the field but everything seems to work without it ^^. I'll paste the error message and investigate a bit further